src/libraries/System.Private.Xml.Linq/tests/XDocument.Common/BridgeHelpers.cs

```diff
@@ -539,42 +539,51 @@ public static void DeleteTestFile(string strFileName)
 
         public static void CreateByteTestFile(string strFileName)
         {
-            FilePathUtil.addStream(strFileName, new MemoryStream());
-            TextWriter tw = new StreamWriter(FilePathUtil.getStream(strFileName));
+            var ms = new MemoryStream();
+            TextWriter tw = new StreamWriter(ms, leaveOpen: true);
```

Comment:
I changed these to  have `using` and then remove the redundant Flushes and Dispose.